### PR TITLE
Add Okta SSO to list of IDP providers from One CLI

### PR DESCRIPTION
**Pull request by sohflp**
Add Okta SSO related features to One CLI:

## Proposed Changes

    - Associate new docker image dnxsolutions/aws-okta-auth
    - Create new command **one auth okta** for authentication with new IDP
    - Create new command under **one auth configure** for Okta IDP parameters configuration

### CODE REVIEW

File: one/commands/auth.py [hunk 1]
```diff
@@ -74,6 +74,29 @@ def azure():
     shutil.move(CLI_ROOT + '/.env', CLI_ROOT + '/secrets')
 
 
+@auth.command(help='Authentication using Okta SSO provider.')
+def okta(auth_image=None):
```
Comment (arthurbdiniz) [inline one/commands/auth.py]: Hi @sohflp, in this case, there is no need to pass `auth_image=None`.

**Review by arthurbdiniz — CHANGES_REQUESTED** [review]
Overall everything looks good. Just a small fix and I think It's ready.
Also, the lint is not passing. Could you please fix It?

```
./one/prompt/idp.py:61:2: W292 no newline at end of file
./one/commands/auth.py:13:128: E501 line too long (141 > 127 characters)
1     E501 line too long (141 > 127 characters)
1     W292 no newline at end of file
```

Nice job!

File: one/commands/auth.py (hunk 1 above)
Comment (sohflp) [reply one/commands/auth.py]: Hey @arthurbdiniz, I removed the parameter as suggested.

**Comment by sohflp** [comment]
> Overall everything looks good. Just a small fix and I think It's ready.
> Also, the lint is not passing. Could you please fix It?
> 
> ```
> ./one/prompt/idp.py:61:2: W292 no newline at end of file
> ./one/commands/auth.py:13:128: E501 line too long (141 > 127 characters)
> 1     E501 line too long (141 > 127 characters)
> 1     W292 no newline at end of file
> ```
> 
> Nice job!

@arthurbdiniz The lint issues are fixed now.

**Pull request merged**